HabboHotel/Users/Profile/ProfileManager.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Dapper;
+using Plus.Database;
+using Plus.HabboHotel.Groups;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Plus.HabboHotel.Users.Profile
+{
+    public class ProfileManager : IProfile
+    {
+        private readonly IGroupManager _groupManager;
+        private readonly IDatabase _database;
+        int friendCount;
+
+        public ProfileManager(IGroupManager groupManager, IDatabase database)
+        {
+            _groupManager = groupManager;
+            _database = database;
+        }
+
+        public async Task <List<Group>> GetGroups(Habbo habbo)
+        {
+            var groups = _groupManager.GetGroupsForUser(habbo.Id);
+            return groups;
+        }
+
+        public async Task<Habbo>? GetProfile(Habbo habbo)
```

Comment:
Why is this a Task? It doesnt do any async calls.
Your editor should give a warning on this. If it doesnt some settings are misconfigured.